api/types/constants.go

```diff
@@ -144,6 +144,9 @@ const (
 	// KindApp is a web app resource.
 	KindApp = "app"
 
+	// KindAppAndIdPServiceProvider represent an App Servers as well as a SAML IdP Service Provider (SAML Application).
```

Comment:
Should we make it clear that this is not a "real" resource that is stored in the backend and is only a "pseudo" resource to provide a common interface to the `ListResources` RPC?